i18n/zh/docusaurus-plugin-content-docs/current/end-user/components/cloud-services/cloud-resources-list.md

```diff
@@ -1,93 +1,136 @@
----
-title: 云资源列表
```

Comment:
pleae still make the Chinese version work

Reply:
Fixed, thanks.